drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -773,6 +773,9 @@ def _mark_complete_and_publish_grade(self):
             self.raw_earned = current_raw_earned
             self._publish_grade(Score(self.raw_earned, self.max_score()))
 
```

Comment:
@e-kolpakov Maybe instead, you could publish the grade even `if current_raw_earned is None or current_raw_earned >= saved_raw_earned:`?